src/Sulu/Component/Content/Types/Link.php

```diff
@@ -23,6 +23,8 @@
 class Link extends SimpleContentType
 {
     public const LINK_TYPE_EXTERNAL = 'external';
+    public const LINK_TYPE_PAGE = 'page';
+    public const LINK_TYPE_MEDIA = 'media';
```

Comment:
`page` and `media` are dynamic link content types. They should not be added here as constant. As they come from a registry and more types are possible. Only the external type is a internal type which we need handle here. Where was it required for you to check for this constants?

Reply:
So your description. The headlessbundle should also not care which provider exist it should just work the same way and use the `LinkProviderPool`.

Reply:
okay, than lets close the PR :)